frontends/open-discussions/src/lib/url.js

```diff
@@ -97,21 +98,37 @@ export const CONTENT_POLICY_URL = "/content_policy/"
 export const SETTINGS_URL = "/settings/"
 export const NOTIFICATION_SETTINGS_URL = "/settings/notifications"
 export const ACCOUNT_SETTINGS_URL = "/settings/account"
-export const PASSWORD_RESET_URL = "/password_reset/"
+export const PASSWORD_RESET_URL = SETTINGS.FEATURES.KEYCLOAK_ENABLED
+  ? "/login/ol-oidc/"
+  : "/password_reset/"
 export const PASSWORD_CHANGE_URL = "/settings/password"
 
 // auth urls
-export const LOGIN_URL = "/login/"
-export const LOGIN_PASSWORD_URL = "/login/password/"
-export const LOGIN_PROVIDER_URL = "/login/external/"
-
-export const REGISTER_URL = "/signup/"
-export const REGISTER_CONFIRM_URL = "/signup/confirm/"
-export const REGISTER_DETAILS_URL = "/signup/details/"
```

Comment:
Keep around and use instead of the ones below if `FEATURE_KEYCLOAK_ENABLED=False`?